samples/dnn/models.yml

```diff
@@ -75,6 +78,25 @@ tiny-yolo-voc:
   height: 416
   rgb: true
   classes: "object_detection_classes_pascal_voc.txt"
+  background_label_id: 0
+  sample: "object_detection"
+
+# Model should be converted to ONNX:
+# from ultralytics import YOLO
+# model = YOLO("yolov8n.pt")
+# model.export(format="onnx")
+yolov8:
+  load_info:
+    url: "https://github.com/ultralytics/assets/releases/download/v0.0.0/yolov8n.pt"
```

Comment:
ONNX link is required here.

Reply:
It's better to rename `model: "yolov8net.onnx"` to `model: "yolov8n.pt"`.

> config for download_models.py contains link to pt, but not onnx. The file is renamed to .onnx as it's defined in yaml config.

I have not seen entry for YOLOv8 in download_models.py except https://github.com/CVHub520/X-AnyLabeling/releases/download/v0.1.0/yolov8n.onnx which is not an official repo. Can we keep a link to `.pt`?

Reply:
Oh, excuse me. I just missed that there is another https://github.com/opencv/opencv/blob/4.x/samples/dnn/download_models.py

Reply:
Replaced to [URL from opencv_extra](https://github.com/opencv/opencv_extra/blob/0c9ec227253c60165b125c7bce313fc2bb4cb592/testdata/dnn/download_models.py#L1008)